### PR TITLE
[DEV-11478] Support different layouts for `Embed`

### PR DESCRIPTION
**Pull request by MohammadxAli**

### CODE REVIEW

File: src/elements/Embed/Embed.scss [hunk 1]
```diff
@@ -6,6 +6,19 @@
...
     margin: $block-spacing-vertical 0;
 
+    &--expanded {
+        @include media-layout-expanded;
+    }
+
+    &--contained {
+        margin-left: auto;
+        margin-right: auto;
+    }
```
Comment (e1himself) [inline src/elements/Embed/Embed.scss:16]: Is this necessary? As far as I understand, embeds were always contained before, and it worked. Right?

**Review by e1himself — APPROVED** [review]

File: src/elements/Embed/Embed.scss (hunk 1 above)
Comment (MohammadxAli) [reply src/elements/Embed/Embed.scss:16]: They had `0px` left and right margin:

https://github.com/prezly/content-renderer-react-js/blob/80dd4bf0aef8a19ce9a90b1ddd8460c0cf8dba03/src/elements/Embed/Embed.scss#L7

Plus, I guess having the `--contained` modifier allows for more customization from outside for people using this, so it's good.

Do you think it makes sense to update the Storybook with the new layouts as well?

Comment (e1himself) [reply src/elements/Embed/Embed.scss:16]: > Do you think it makes sense to update the Storybook with the new layouts as well?

Absolutely! :+1:

**Review by e1himself — APPROVED** [review]
:+1:

**Pull request merged**